components/log-viewer-webui/server/.env

```diff
@@ -1,3 +1,8 @@
 CLIENT_DIR=../client/dist
+IR_DATA_DIR=../../../build/clp-package/var/data/ir
+LOG_VIEWER_DIR=../yscope-log-viewer/dist
```

Comment:
@Henry8192 Kindly update here once your `yscope-log-viewer` integration PR is ready.